### DISCUSSION TITLE
Show only columns selected in Columns Sheet

### DISCUSSION THREAD

**Discussion by ECon87** [saulpw/visidata#1326] [Q&A] [answered]
This is probably easy to do but I can't figure it out. I am in the Columns Sheet and regex selected the 2 columns I want to keep.

Is there a way to hide all other columns and create a new sheet with the 2 columns?

**Comment by saulpw** [comment, marked as answer]
Hi @ECon87, you can do `gt` to toggle the selection and then `g-` to hide all selected columns.  If you want to create a new sheet, use `g"` first on the source sheet to copy it.

**Reply by ECon87** [reply]
@saulpw Thank you. This worked like a charm.

Thank you for this great tool also.
